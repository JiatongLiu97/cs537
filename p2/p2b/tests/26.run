./mysh tests/26.in
