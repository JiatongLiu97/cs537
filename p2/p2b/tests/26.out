/bin/ls tests/p2a-test > tests-out/output26
/bin/cat tests-out/output26
/bin/echo this should be in the file > tests-out/output26
/bin/cat tests-out/output26
this should be in the file
/bin/echo hello
hello
